charts/clickhouse/values.yaml

```
@@ -217,340 +154,411 @@
-  - "172.16.0.0/12"
-  - "192.0.0.0/24"
-  - "198.18.0.0/15"
-  - "192.168.0.0/16"
-
-persistence:
-  # -- Enable data persistence using PVC.
+# ClickHouse Instance Configuration
+clickhouseInstance:
+  # -- Whether to install clickhouse instance. If false, only the operator will be installed
   enabled: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)